### PR TITLE
feat: Change the props, set/get config

### PR DESCRIPTION
**Pull request by Blu-J**
## About

Need to have end to end get/set config
properties

## Proof

Showing it working
https://user-images.githubusercontent.com/2364004/170088448-8e85d60c-f5c9-4dc8-9d25-5dc93fe353db.mp4

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1 +1,3 @@
-{}
+{
+  "deno.enable": true
+}
```
Comment (chrisguida) [inline .vscode/settings.json:3]: Cool, I guess this enables syntax highlighting? Useful if so

File: docker_files/health-check.sh [hunk 2]
```diff
@@ -1,9 +1,10 @@
 #!/bin/sh
 
 check_web(){
-    curl --silent --fail localhost:8384/ &>/dev/null
+    curl -u $(jq -r '.username' /root/config.json):$(jq -r '.password' /root/config.json) --silent --fail localhost:8384/ &>/dev/null
```
Comment (chrisguida) [inline docker_files/health-check.sh:4]: was this failing before? or is it something that was added to the new version of syncthing?

**Review by chrisguida — APPROVED** [review]
Sanity checked, works. Code looks good.

File: docker_files/health-check.sh (hunk 2 above)
Comment (Blu-J) [reply docker_files/health-check.sh:4]: Added to make sure we are failing if we can't login with the set properties. It was passing when I thought it should fail. I wasnn't creating the right passwords + such. Now I'm checking that.

**Pull request merged**